### DISCUSSION TITLE
CSV Import - Data Load Setting - "Open table editor on finish" not working consistently

### DISCUSSION THREAD

**Discussion by scokna4gh** [dbeaver/dbeaver#21055] [Q&A]
I import CSV files regularly through the "Import Data" tool in DBeaver. I prefer to review the data after the import is completed, so I turn on the Data Load setting labeled "Open table editor on finish". This opens the table editor after the import completes...until recently. When I upgraded to DBeaver 23.1.5, the table editor does not open consistently after a CSV file import completes. Is there a new setting that I need to turn on or is this a bug?

Here is a screenshot of the Data Load settings that I use regularly:
![image](https://github.com/dbeaver/dbeaver/assets/134745719/da5ad3a4-d80d-47eb-8883-f78cc1a317e0)

Any assistance is appreciated.  :-)

**Comment by E1izabeth** [comment]
It looks more like a bug as we haven't changed this behavior. However I couldn't reproduce the issue. Could you pay attention to the details when it's happening maybe you can figure out the concrete steps to reproduce?

**Comment by LonwoLonwo** [comment]
Hello @scokna4gh 

Could you please check the latest DBeaver version? https://dbeaver.io/download/
